internal/admin/admin.go

```diff
@@ -697,6 +697,10 @@ func (adm *Admin) updateLogStream(ctx context.Context, lsid types.LogStreamID, p
 	newLSDesc := proto.Clone(oldLSDesc).(*varlogpb.LogStreamDescriptor)
 	newLSDesc.Replicas[popIdx] = &pushedReplica
 
+	if !adm.hasSealedReplica(ctx, newLSDesc) {
+		return nil, status.Errorf(codes.FailedPrecondition, "update log stream: does not have sealed replica")
```

Comment:
How about a more concise message like "update log stream: no sealed replica" instead of "does not have sealed replica"?